src/main/kotlin/org/koitharu/kotatsu/parsers/site/foolslide/en/AssortedScans.kt

```diff
@@ -0,0 +1,141 @@
+package org.koitharu.kotatsu.parsers.site.foolslide.en
+
+
+import kotlinx.coroutines.coroutineScope
+import org.jsoup.nodes.Document
+import org.koitharu.kotatsu.parsers.MangaLoaderContext
+import org.koitharu.kotatsu.parsers.MangaSourceParser
+import org.koitharu.kotatsu.parsers.model.*
+import org.koitharu.kotatsu.parsers.site.foolslide.FoolSlideParser
+import org.koitharu.kotatsu.parsers.util.*
+import java.util.ArrayList
+
+
+@MangaSourceParser("ASSORTEDSCANS", "AssortedScans", "en")
+internal class AssortedScans(context: MangaLoaderContext) :
+	FoolSlideParser(context, MangaSource.ASSORTEDSCANS, "assortedscans.com", 56) {
+
+	override val listUrl = "reader/"
+	override val pagination = false
+	override val selectInfo = "div.#series-info"
+
+	override suspend fun getListPage(
+		page: Int,
+		query: String?,
+		tags: Set<MangaTag>?,
+		sortOrder: SortOrder,
+	): List<Manga> {
+
+		val doc = if (!query.isNullOrEmpty()) {
+
+			val url = buildString {
+				append("https://$domain/$searchUrl")
```

Comment:
If you already use a `StringBuilder` it might be better to avoid concatenation

```suggestion
				append("https://")
				append(domain)
				append('/')
				append(searchUrl)
```